content/zh/blogs/cilium-kubesphere-kubernetes.md

```diff
@@ -0,0 +1,184 @@
+---
+title: '使用 Cilium 作为网络插件部署 Kubernetes + KubeSphere'
+tag: 'Kubernetes,KubeSphere,Cilium'
+keywords: 'Kubernetes, KubeSphere, Cilium, 网络插件'
+description: 'Cilium 是一个用于容器网络领域的开源项目，主要是面向容器而使用，用于提供并透明地保护应用程序工作负载（如应用程序容器或进程）之间的网络连接和负载均衡。本文主要介绍如何使用 Cilium 作为网络插件部署 Kubernetes 和 KubeSphere。'
+createTime: '2021-06-04'
+author: '姚锐'
+snapshot: 'https://pek3b.qingstor.com/kubesphere-community/images/Cilium-KubeSphere-Kubernetes .png'
+---
+
+## Cilium 简介
+Cilium 是一个用于容器网络领域的开源项目，主要是面向容器而使用，用于提供并透明地保护应用程序工作负载（如应用程序容器或进程）之间的网络连接和负载均衡。
+
+Cilium 在第 3/4 层运行，以提供传统的网络和安全服务，还在第 7 层运行，以保护现代应用协议（如 HTTP, gRPC 和 Kafka）的使用。 Cilium 被集成到常见的容器编排框架中，如 Kubernetes 和 Mesos。
+
+Cilium 的底层基础是 BPF，Cilium 的工作模式是生成内核级别的 BPF 程序与容器直接交互。区别于为容器创建 overlay 网络，Cilium 允许每个容器分配一个 IPv6 地址（或者 IPv4 地址），使用容器标签而不是网络路由规则去完成容器间的网络隔离。它还包含创建并实施 Cilium 规则的编排系统的整合。
+
+![](https://pek3b.qingstor.com/kubesphere-community/images/cilium.png)
+
+> 以上简介来源于 oschina
+
+## 系统要求
+Linux Kernel >= 4.9.17
+更多信息请查看 [Cilium 系统要求](https://docs.cilium.io/en/v1.9/operations/system_requirements/)
+
+## 环境
+以一台 Ubuntu Server 20.04.1 LTS 64bit 为例
+| name | ip | role |
+| --- | ---| --- |
```

Comment:
I see the mistaken format in preview.
![image](https://user-images.githubusercontent.com/40452856/125582294-e3957e07-1a8a-4295-8551-68a49ae5ea13.png)